### PR TITLE
Geogeek au.GitHub.io v2

### PR DESCRIPTION
**Pull request by kd112**

### CODE REVIEW

File: _layouts/Default.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  <div class="container">
+    {% if site.data.events.events %}
+      {% for event in site.data.events.events %}
+        <div class="row justify-content-center">
+          <div class="col-lg-8 text-left mb-3">
+            <h5 class="text-white mt-0">
+              {{event.name}}
+            </h5>
+            <div class="text-white-50 pl-5">
+              {% if event.description %}{{event.description}}</br></br>{% endif %}
```
Comment (samwilson) [inline _layouts/Default.html:128]: `</br>` should be `<br />`

File: _layouts/Default.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+      ga('send', 'pageview');
+      </script>
+  </head>
+  <body id="page-top">
+
+    <!-- Capture the Footer Attribution HTML in a variable -->
+    {%capture attribution%}
+    <div class="row justify-content-center">
+        <div class="col-lg-8 col-lg-offset-2 text-center">
+            <a href="https://spacecubed.com/" target="_blank"><img src="assets/img/logospacecubed.png" alt="Spacecubed logo."></a>
```
Comment (samwilson) [inline _layouts/Default.html:69]: The `<div class="col-lg-8 col-lg-offset-2 text-center">` element should be closed here (I think?).

**Review by samwilson — COMMENTED** [review]
Looks terrific!!

A few thoughts in addition to the other comments:

* We should keep the LICENSE file.
* Rename `Readme.md` back to `README.md`?
* Do we need to keep `oldSite.html`? It's in the Git history anyway.
* The `/geogeeks.ics` URL should maybe keep working (I'm happy to figure out how to make this build itself from `events.yml` but for now it can just be static I think).

File: _layouts/Default.html (hunk 1 above)
Comment (themissingbracket) [reply _layouts/Default.html:128]: yes, the oldSite.html can be removed,  I had kept it more for a quick reference for what the old site had interms of content, and general layout

Comment (themissingbracket) [reply _layouts/Default.html:128]: I will add the license and the rename the readme file later

Comment (kd112) [reply _layouts/Default.html:129]: PS, that is my other github profile

**Pull request closed without merging**